querybook/webapp/lib/utils/embed.ts

```diff
@@ -0,0 +1,4 @@
+const EMBED_PATH_STRING = 'embedded';
+
+export const isEmbedPath = (path: string): boolean =>
```

Comment:
imo this is a really one off case that we don't need a file for it, since it could be done with `window.location.pathname.includes(EMBED_PATH_STRING)`

Reply:
Ya, I was trying to avoid adding logic into the reducer, but you're right, it's pretty bespoke so it's probably fine to leave here.

Can always refactor later if we add different embedding capabilities.